src/include/daos_fs.h

```diff
@@ -1122,6 +1121,27 @@ dfs_removexattr(dfs_t *dfs, dfs_obj_t *obj, const char *name);
 int
 dfs_listxattr(dfs_t *dfs, dfs_obj_t *obj, char *list, daos_size_t *size);
 
+
+enum {
+	DFS_CHECK_PRINT		= (1 << 0),
+	DFS_CHECK_REMOVE	= (1 << 1),
+	DFS_CHECK_LINK_LF	= (1 << 2),
```

Comment:
Better documentation would make sense here, having reviewed the PR I know what "LF" means but if someone just opens the header file without context it won't be so obvious.